1.8/administration/installing/custom/cli.md

```diff
@@ -160,6 +160,11 @@ The DC/OS installation creates these folders:
     - 8.8.8.8
     ssh_port: 22
     ssh_user: <username>
+    use_proxy: 'true'
+    http_proxy: http://<your_http_proxy>/
+    https_proxy: https://<your_https_proxy>/
+    no_proxy: 
+    - '*.int.example.com' 
```

Comment:
For the people who deal with proxies for their environment I think they'll have a pretty good idea what all these mean coming out of the box. I think just describing them in the "full list of configuration options" should be enough